node/primitives/src/approval.rs

```diff
@@ -77,6 +77,38 @@ pub enum AssignmentCertKind {
 	},
 }
 
+/// Certificate is changed compared to `AssignmentCertKind`:
+/// - introduced RelayVRFModuloCompact
+#[derive(Debug, Clone, Encode, Decode, PartialEq, Eq)]
+pub enum AssignmentCertKindV2 {
+	/// An assignment story based on the VRF that authorized the relay-chain block where the
+	/// candidate was included combined with a sample number.
+	///
+	/// The context used to produce bytes is [`RELAY_VRF_MODULO_CONTEXT`]
+	RelayVRFModulo {
+		/// The sample number used in this cert.
+		sample: u32,
```

Comment:
In essence, this sample number goes away.  We'll only have one sample which generates all ouputs.